touchforms/formplayer/static/formplayer/script/deprecate/entrycontrols_type.js

```diff
@@ -355,9 +355,9 @@ function MultiSelectEntry (args) {
     return function () {
       var cbox = $('#ch-' + i);
       if (cbox.is(':checked')) {
-        cbox.removeAttr('checked');
+        cbox.prop('checked', false);
```

Comment:
i don't think entrycontrols_type.js or typeforms-ui.js is used anymore, but good find! i was scared to delete it so i moved it to the deprecate folder